composer.json

```diff
@@ -24,6 +24,7 @@
     "require": {
         "php": "^8.1",
         "ext-mongodb": "^1.15",
+        "composer-runtime-api": "^2.0.0",
```

Comment:
If this is now required, should you remove the `class_exists(InstalledVersions::class)` check in [Connection.php](https://github.com/mongodb/laravel-mongodb/blob/b55393ca2cbd9eb68d934089e610390eb5147549/src/Connection.php#L327)?

Reply:
Of course.